requirements/test_redirects.txt

```diff
@@ -0,0 +1,9 @@
+-r dev.txt
```

Comment:
Do we need many packages from `dev.txt`? Does it make sense to add them in `test.txt` and don't include `dev.txt` here?